scripts/unsupported/combine_tracks_postprocessing_cnv/combine_tracks.wdl

```diff
@@ -0,0 +1,621 @@
+# A postprocessing workflow for GATK CNV.
+#
+# THIS CANNOT BE RUN IN TUMOR-ONLY MODE.  A MATCHED NORMAL IS REQUIRED.
+#
+# Internally, we have shown that this workflow improves sensitivity and precision.  Additionally, this workflow is quite
+#  cheap to run.  Note that auxiliary files are required and without these exact files, performance may suffer.
+#
+# UNSUPPORTED and we may modify this WDL before we fully support it.
+#
+# This will also generate absolute (https://software.broadinstitute.org/cancer/cga/absolute) compatible files.
+#  Currently, the balanced-segment calling is extremely ham-fisted.
+#
+# This workflow:
+#  - Does conversion of output seg files to a format compatible with IGV.  This will be removed soon, since the GATK tools
```

Comment:
Why not just remove this stuff now?

Reply:
For now, I need to be able to run this workflow on old runs of GATK CNV due to restrictions on collaborators.

I filed issue https://github.com/broadinstitute/gatk/issues/5281 to update the WDL